docs/reports/FINANCEFEE-351-mes-cost-calibration.md

```
@@ -0,0 +1,26 @@
+HOST=gpu-laptop FINANCEFEE-351 — Real MES Cost Calibration
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)